ui/desktop/tests/acceptance/projects/sessions/session-test.js

```diff
@@ -160,7 +164,29 @@ module('Acceptance | projects | sessions | session', function (hooks) {
     });
 
     await visit(urls.target);
-    await click('[data-test-target-detail-connect-button]');
+    await click(TARGET_CONNECT_BUTTON);
+
+    assert.strictEqual(currentURL(), urls.session);
+  });
+
+  test('visiting a session that does not have read permissions but a successful connect', async function (assert) {
+    assert.expect(2);
+    // verifies second call to sessions/:id is also a 403
+    QUnit.onUncaughtException = (err) => {
```

Comment:
We've talked about this offline but I verified that this will set it globally so we need to reset this back to what it was at the end of this test.